src/components/KRadio/KRadio.vue

```diff
@@ -11,105 +12,219 @@
       @click="handleClick"
     >
     <span
-      v-if="hasLabel"
+      v-if="isTypeDefault && hasLabel"
       class="k-radio-label"
     >
       <slot>{{ label }}</slot>
     </span>
     <div
-      v-if="hasLabel && (description || $slots.description)"
+      v-if="isTypeDefault && hasLabel && hasDescription"
       class="k-radio-description"
     >
       <slot name="description">{{ description }}</slot>
     </div>
+    <div v-if="!isTypeDefault && hasLabel">
+      <slot />
```

Comment:
nit: can you add the default name here so it's obvious?
```suggestion
      <slot name="default" />
```

Reply:
Sure, done